app/application/template.hbs

```diff
@@ -1,7 +1,7 @@
 {{title (t 'general.OSF')}}
 <div class="Application">
     {{osf-navbar signupUrl=signupUrl loginAction=(action 'login')}}
-    <div id="secondaryNavbar"></div>
+    <div id="{{config.OSF.secondaryNavbarId}}"></div>
```

Comment:
`config` isn't set here, you need to get it in the controller. Right now ember-wormhole fails to find its outlet and just renders in place, which, turns out, looks good.